app/models/case.rb

```diff
@@ -2,9 +2,21 @@
 
 # This is the main model of the application. Each death is a case.
 # TODO: Lots & lots of refactoring
+# rubocop:disable Metrics/ClassLength
 class Case < ApplicationRecord
   # TODO: Clean up relationship section
```

Comment:
This todo is from 2016. I think that that comment can be removed.

Reply:
Ok, I'll remove both of them in the next PR touching this file.